util/cinc-install.sh

```diff
@@ -468,8 +468,14 @@ elif test -f "/etc/Eos-release"; then
 elif test -f "/etc/redhat-release"; then
   platform=`sed 's/^\(.\+\) release.*/\1/' /etc/redhat-release | tr '[A-Z]' '[a-z]'`
   platform_version=`sed 's/^.\+ release \([.0-9]\+\).*/\1/' /etc/redhat-release`
-
-  if test "$platform" = "xenserver"; then
+  
+  if test "$platform" = "rocky linux"; then
```

Comment:
How is this change related to AL2023?

Reply:
Initially when I added changes in cinc-install.sh it was for both Al2023 and Rocky linux, but after a team discussion we decided to not upgrade for rocky in cinc-install.

However, I had already upgraded cinc-install script as part of https://github.com/aws/aws-parallelcluster-cookbook/pull/2712

Reply:
So we can remove this piece of code for rocky

Reply:
Removed